### ISSUE TITLE
Consider eliminating CGSize version of measure and calculate methods

### ISSUE THREAD

**Issue by nguyenhuy** [facebook/AsyncDisplayKit#557]
Follow up issue after #468:

Once the box model is stable and widely adopted, it's interesting to think about what should be done to -measure: and -calculateSizeThatFits: methods. They are the legacies of the old manual layout model. In the new box model, manual layout can be achieved by returning a manually-constructed ASLayout (via -calculateLayoutThatFits:), or even by returning an ASStaticLayoutSpec (via -layoutSpecThatFits:). And unlike -calculateSizeThatFits, if one of the two alternative approaches is used, the default implementation of -layout will work properly. So perhaps we should remove or deprecate these CGSize methods?

**Comment by appleguy** [comment]
@nguyenhuy - moving this to Ship. :)

The community is planning an exciting long term road map for the project and getting organized around how to deliver these feature requests. We are moving to a new issue tracker with more sophisticated planning capabilities (check out www.realArtists.com and their new product: Ship!). 

If you are interested in helping contribute to this component or any other, don’t hesitate to send me an email at AsyncDisplayKit@gmail.com. If you would like to contribute for a few weeks, I can also add you to our Ship bug tracker so that you can see what everyone is working on and actively coordinate with us. 

As always, keep filing issues and submitting pull requests here on Github and we will only move things to the new tracker if they require long term coordination.
